test/spec/modules/outconBidAdapter_spec.js

```diff
@@ -0,0 +1,104 @@
+import { expect } from 'chai';
+import { spec } from '../../../modules/outconBidAdapter';
+
+describe('outconBidAdapter', function () {
+  describe('bidRequestValidity', function () {
+    it('Check the bidRequest with pod param', function () {
+      expect(spec.isBidRequestValid({
+        bidder: 'outcom',
```

Comment:
To ensure consistency - can you please fix these `bidder` values through your tests to use `outcon`?